### DISCUSSION TITLE
Collapse laterally

### DISCUSSION THREAD

**Discussion by juersierra** [saadeghi/daisyui#1627] [Q&A] [answered]
Is there a way to make the extending of the collapse component goes to the right (or left) instead of downside? The dropdown components has similar option to what im asking (making it dropdown the left, right, up, down).
Also wondering if you can make it expand while hover.
Thanks!

**Comment by saadeghi** [comment, marked as answer]
I have no plans for adding those modifications because I don't think the majority of users would need them.  
But you can always add your own CSS to modify the style as you want.
